src/server.rs

```diff
@@ -15,113 +14,150 @@ use crate::configuration::Configuration;
+
+        match req.origin.headers.get::<Authorization<Bearer>>() {
+            // TODO: get secret from configuration file
+            Some(header) => match decode::<Claims>(&header.token, "0cccdfddd23f9f740c9620a094daf1b260436059924cec32622f0e7ebc99bbe5".as_ref(), &Validation::default()) {
+                Ok(_bearer) => match self.http_verb {
+                    Method::Head => match req.param("key") {
+                        Some(key) => match &store.get(key.to_string()) {
+                            Some(_) => {
+                                res.set(StatusCode::Ok);
+                                res.send("")
+                            },
+                            None => {
+                                res.set(StatusCode::NotFound);
+                                res.send("")
+                            }
+                        },
+                        None => {
+                            res.set(StatusCode::BadRequest).set(MediaType::Json);
+                            res.send(serde_json::to_string_pretty(&ErrorMessage { code: 102, message: "Missing key parameter.", details: None }).unwrap())
+                        }
                     },
-                    None => {
-                        res.set(StatusCode::NotFound);
-                        res.send("")
-                    }
-                },
-                _ => {
-                    res.set(StatusCode::BadRequest).set(MediaType::Json);
-                    res.send(serde_json::to_string_pretty(&ErrorMessage { message: "Missing key parameter.".to_string(), details: None }).unwrap())
-                }
-            },
-            Method::Put => {
-                if body_size == 0 {
-                    res.set(StatusCode::BadRequest).set(MediaType::Json);
-                    return res.send(serde_json::to_string_pretty(&ErrorMessage { message: "Missing request body.".to_string(), details: None }).unwrap());
-                }
-                match req.param("key") {
-                    Some(key) => if buffer.len() < 7340032 {
-                        store.set(key.to_string(), buffer);
-                        res.set(StatusCode::Ok);
-                        res.send("")
-                    } else {
-                        res.set(StatusCode::BadRequest).set(MediaType::Json);
-                        res.send(serde_json::to_string_pretty(&ErrorMessage { message: "The maximum allowed value size is 7 Mb.".to_string(), details: None }).unwrap())
+                    Method::Put => {
+                        if body_size == 0 {
+                            res.set(StatusCode::BadRequest).set(MediaType::Json);
+                            return res.send(serde_json::to_string_pretty(&ErrorMessage { code: 101, message: "Missing request body.", details: None }).unwrap());
+                        }
+                        match req.param("key") {
+                            Some(key) => if buffer.len() < 7340032 {
```

Comment:
This max value size must be a global constant.

Reply:
No, it needs to be a configuration parameter, but like the previous comment, it will be done here: https://github.com/orgs/lucid-kv/teams/lucid-team/discussions/3